frontend/src/employee-frontend/components/holiday-term-periods/api.ts

```diff
@@ -98,6 +98,26 @@ export async function getPreschoolTermsResult(): Promise<PreschoolTerm[]> {
     .then((res) => res.data.map(deserializePreschoolTerm))
 }
 
+export async function getPreschoolTerm(termId: UUID): Promise<PreschoolTerm> {
```

Comment:
these could now be replaced by the auto-generated ones, but I can also do that together with refactoring other places

Reply:
Now I removed all the manually made clinet calls and replaced in the queries.ts all the api calls to use api-clients that are generated from codegen. Good to know that the generated clients are now ready for use.